index.html

```diff
@@ -75,18 +76,163 @@ <h1>ADEMOLA OWOEYE</h1>
                 <i class="fa-brands fa-twitter"></i></li>
                 <li class="M-icon">
                 <i class="fa-solid fa-m"></i></li>
-                </div>
+                </ul>
+            </div>
 
-               <div class="work-down-arrow">
+                <div class="work-down-arrow"> 
                 <div class="icon">
                     <i class="fa-solid fa-arrow-down"></i>
                     </div>
-                <div class="work"> work </div>
-              </div>
+                   <div class="wor" > work </div> 
+                 </div> 
+    </div>            
+</section>
+        
 
-            </div>
 
+
+<section class="work">
+      
+    <div class="title-work">
+     <h2 class="recent">MY RECENT</h2>
+     <h2> WORK </h2>
     </div>
-</section>
+
+    <div class="line">.</div>
+ 
+   <img class="image" src="Screenshot(349).png" alt="white">
+
+   <h3>Multi-Post Stories</h3>
+
+   <p class="text">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
+
+<ul class="skil">
+   <li class="skill">css</li>
+   <li class="vector">.</li>
+   <li class="skill">html</li>
+   <li class="vector">.</li>
+   <li class="skill">bootstrap</li>
+   <li class="vector">.</li>
+   <li class="skill">ruby</li>
+</ul>
+
+   <button class="btn" type="button"> See Project </button>
+
+</section> 
+
+<section class="work2">
+ 
+   <img class="image" src="Screenshot(349).png" alt="white">
+
+   <h3>Multi-Post Stories</h3>
+
+   <p class="text">  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text. </p>
+
+<ul class="skil">
+   <li class="skill">css</li>
+   <li class="vector">.</li>
+   <li class="skill">html</li>
+   <li class="vector">.</li>
+   <li class="skill">bootstrap</li>
+   <li class="vector">.</li>
+   <li class="skill">ruby</li>
+</ul>
+
+   <button class="btn btn-p" type="button">  See Project </button>
+
+</section> 
+ 
+<section class="work2">
```

Comment:
[OPTIONAL] 

-  I think you should consider replacing this `section` tag with a `div` tag because each project forms part of the works section. This is necessary to make your code semantic. You can read more on the usage of section and div [here](https://thisthat.dev/div-vs-section/.)👍 ;